### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by NoooneyDude**
- Fixes some compiler warnings.
- Namespaces a few classes.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Effects/CustomEffectBehaviour.cs [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 public class CustomEffectBehaviour : MonoBehaviour
 {
-	public ParticleSystem particleSystem;
+	public new ParticleSystem particleSystem;
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Effects/CustomEffectBehaviour.cs]: why new?

Comment (NoooneyDude) [reply UnityProject/Assets/Scripts/Effects/CustomEffectBehaviour.cs]: MonoBehaviour already has a `particleSystem` member. So we just ignore it and use our (assumed serialized) one instead. I could just rename the variable.

Comment (corp-0) [reply UnityProject/Assets/Scripts/Effects/CustomEffectBehaviour.cs]: Renaming is probably the best solution

**Review by ThatDan123 — DISMISSED** [review]

**Comment by corp-0** [comment]
merge conflict :c

**Review by corp-0 — APPROVED** [review]

**Review by ThatDan123 — APPROVED** [review]

**Pull request merged**